test/ts/migrations/5_authorize_contract_interactions.ts

```diff
@@ -0,0 +1,104 @@
+// External Libraries
+import * as chai from "chai";
+
+// Utils
+import ChaiSetup from "../test_utils/chai_setup";
+
+// Wrappers
+import { SimpleInterestTermsContractContract as SimpleInterestTermsContract } from "../../../types/generated/simple_interest_terms_contract";
+import { CollateralizedSimpleInterestTermsContractContract as CollateralizedTermsContract } from "../../../types/generated/collateralized_simple_interest_terms_contract";
+import { CollateralizerContract as Collateralizer } from "../../../types/generated/collateralizer";
+import { DebtKernelContract } from "../../../types/generated/debt_kernel";
+import { DebtTokenContract } from "../../../types/generated/debt_token";
+import { DebtRegistryContract } from "../../../types/generated/debt_registry";
+import { TokenTransferProxyContract } from "../../../types/generated/token_transfer_proxy";
+import { RepaymentRouterContract } from "../../../types/generated/repayment_router";
+
+ChaiSetup.configure();
+
+const expect = chai.expect;
+
+contract("Migration #5: Authorizing Contract Interactions", async (ACCOUNTS) => {
+    const CONTRACT_OWNER = ACCOUNTS[0];
+    const TX_DEFAULTS = { from: CONTRACT_OWNER, gas: 4000000 };
+
+    let collateralizer: Collateralizer;
+    let kernel: DebtKernelContract;
+    let debtToken: DebtTokenContract;
+    let registry: DebtRegistryContract;
+    let proxy: TokenTransferProxyContract;
+
+    describe("Deployment", () => {
+        before(async () => {
+            collateralizer = await Collateralizer.deployed(web3, TX_DEFAULTS);
+            debtToken = await DebtTokenContract.deployed(web3, TX_DEFAULTS);
+            kernel = await DebtKernelContract.deployed(web3, TX_DEFAULTS);
+            registry = await DebtRegistryContract.deployed(web3, TX_DEFAULTS);
+            proxy = await TokenTransferProxyContract.deployed(web3, TX_DEFAULTS);
+        });
+
+        it("should authorize `DebtToken` to insert into the registry", async () => {
+            const approved = await registry.getAuthorizedInsertAgents.callAsync();
+
+            expect(approved).to.include(debtToken.address);
+        });
+
+        it("should authorize `DebtToken` to edit the registry", async () => {
+            const approved = await registry.getAuthorizedEditAgents.callAsync();
+
+            expect(approved).to.include(debtToken.address);
+        });
+
+        it("should authorize the kernel to mint and broker debt tokens", async () => {
```

Comment:
Can get rid of the `broker` statement here -- that feature is no longer relevant.